cmd/share-secret/ui/assets/index.html

```diff
@@ -7,10 +7,10 @@
   <body>
         <h1>Share-Secret</h1>
 
-        <form id="secret-form">
+        <form>
           <label for="secret">Secret Text:</label>
           <br>
-          <input type="text" id="secret" name="secret" required  minlength="1" maxlength="200" size="50" />          
+          <textarea id="secret" name="secret" required  minlength="1" maxlength="400" rows="10" cols="50"></textarea>
```

Comment:
Why the limit? You're making a POST there are no limits in that. Remove it later